src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -49,6 +49,7 @@ namespace Akka.Actor
         public const int UndefinedUid = 0;
         public ActorCell(Akka.Actor.Internal.ActorSystemImpl system, Akka.Actor.IInternalActorRef self, Akka.Actor.Props props, Akka.Dispatch.MessageDispatcher dispatcher, Akka.Actor.IInternalActorRef parent) { }
         protected Akka.Actor.ActorBase Actor { get; }
+        public Akka.Actor.Internal.IChildrenContainer ChildrenContainer { get; }
```

Comment:
Needed to make `ChildrenContainer` visible as part of the `ICell` interface; it's like this on the JVM, but also necessary for the `ActorsLeakSpec`